library/alt

```diff
@@ -4,27 +4,39 @@ Maintainers: ALT Linux Team Cloud <alt-cloud@altlinux.org> (@alt-cloud),
              Mikhail Gordeev <obirvalger@altlunux.org> (@obirvalger)
 GitRepo: https://github.com/alt-cloud/docker-brew-alt.git
 
-Tags: p9, latest
```

Comment:
Did you want to keep the `latest` tag to now point to `p10`?

Reply:
Yes, latest should point to p10 from now.